docs/src/pages/guides/migration-v4/migration-v4.md

```diff
@@ -994,6 +994,26 @@ You can use the [`collapse-rename-collapsedheight` codemod](https://github.com/m
 
   You can use the [`use-transitionprops` codemod](https://github.com/mui-org/material-ui/tree/HEAD/packages/material-ui-codemod#use-transitionprops) for automatic migration.
 
+- Default `anchorOrigin.vertical` is changed
+
+  ```diff
+  {
+    - vertical: 'bottom',
+    + vertical: 'top',
+  }
+  ```
+
+### MenuItem
+
+- the `ListItem` inheritance is replaced by `ButtonBase`. ClassNames related to "MuiListItem-\*" are removed and theming `ListItem` is no longer affecting `MenuItem`.
+
+  ```diff
+  - <li className="MuiButtonBase-root MuiListItem-root MuiMenuItem-root">
+  + <li className="MuiButtonBase-root MuiMenuItem-root">
+  ```
+
+- prop `disabledGutters` is removed.
```

Comment:
Active voice.

```suggestion
- Remove the `disabledGutters` prop.
```

Reply:
Are you sure we need to remove this prop? Would it make sense to keep it in to minimize BCs?

Reply:
I will bring it back.